modules/metrics/mule-metrics-exporter-impl/pom.xml

```diff
@@ -54,6 +54,20 @@
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-exporter-otlp</artifactId>
             <version>${opentelemetryVersion}</version>
+            <exclusions>
+                <!-- excluded to prevent a split-package with okio-jvm -->
+                <exclusion>
+                    <groupId>com.squareup.okio</groupId>
+                    <artifactId>okio</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <!-- Added back because this was brought transitively by okio -->
+        <dependency>
+            <groupId>com.squareup.okio</groupId>
+            <artifactId>okio-jvm</artifactId>
+            <version>3.2.0</version>
```

Comment:
Have you verified that this is the last version corresponding to the update in the otel pom?

Reply:
Yes it's 3.2.0